test/unit/migrate/migrator.js

```diff
@@ -71,3 +71,34 @@ describe('Migrator.loadExtensions', function () {
   });
 
 });
+
+describe('Migrator.validateMigrationList', function () {
+  var config = {
```

Comment:
since this is identical to the config in another test, maybe that one should be extracted higher and reused?